mlflow/store/tracking/sqlalchemy_store.py

```diff
@@ -1548,6 +1549,11 @@ def start_trace(
             )
 
             trace_info.tags = [SqlTraceTag(key=k, value=v) for k, v in tags.items()]
+            trace_info.tags.append(
+                SqlTraceTag(
+                    key=MLFLOW_ARTIFACT_LOCATION, value=self._get_artifact_location(experiment_id)
```

Comment:
The artifact location needs to align with FileStore:

https://github.com/mlflow/mlflow/blob/2f22db02f671fe31beb8406e8bbde251038e14d0/mlflow/store/tracking/file_store.py#L1341-L1347

Reply:
A duplicate of https://github.com/mlflow/mlflow/pull/12102/files#r1609532663